src/index.js

```diff
@@ -0,0 +1,39 @@
+// import _ from "lodash";
+import './style.css';
+// import Icon from './images/cheched.jpg';
+
+const tasks = [
+  { description: 'Task 1', completed: false, index: 1 },
+  { description: 'Task 2', completed: true, index: 2 },
+  { description: 'Task 3', completed: false, index: 3 },
+];
+
+function renderTasks() {
+  const listElement = document.querySelector('.list');
+
+  const inputEl = document.createElement('input');
+  inputEl.classList.add('input');
+  inputEl.placeholder = 'Add your task here';
+  listElement.appendChild(inputEl);
+
+  tasks.forEach((task) => {
+    const listItem = document.createElement('li');
+    listItem.textContent = task.description;
+    listItem.classList.add(task.completed ? 'completed' : 'not-completed');
+    listElement.appendChild(listItem);
+
+    const span = document.createElement('div');
+    span.classList.add('close');
```

Comment:
- [ ] Great work making your list structure for your todo list. You've done a great job. However, there's one thing you need to change. Kindly replace the `close` icon on your template with `vertical three dots.` in order to match what is on the wireframe provided.

### Your Design
![your-design](https://github.com/Umuhan/to-do-list/assets/26597432/5baf5a63-5cde-46b3-b04e-76d750735035)

### Wireframe
![wireframe](https://github.com/Umuhan/to-do-list/assets/26597432/fa519638-b8e2-4444-96c2-05d8d20af10d)

Reply:
done.